internal/rukpak/util/labels.go

```diff
@@ -1,6 +1,10 @@
 package util
 
 const (
-	CoreOwnerKindKey = "core.rukpak.io/owner-kind"
-	CoreOwnerNameKey = "core.rukpak.io/owner-name"
+	CoreOwnerKindKey          = "core.clusterextension.io/owner-kind"
+	CoreOwnerNameKey          = "core.clusterextension.io/owner-name"
+	ResolvedBundleCatalogID   = "core.clusterextension.io/catalog-id"
+	ResolvedbundlePackageName = "core.clusterextension.io/package-name"
+	ResolvedbundleName        = "core.clusterextension.io/bundle-name"
+	ResolvedbundleVersion     = "core.clusterextension.io/bundle-version"
```

Comment:
I don't believe we "own" the `clusterextension.io` domain, but this is a PoC...

Reply:
+1 these should definitely be updated before 1.0.0

I'd suggest `olm.operatorframework.io/*`

Reply:
Also a few nits on naming:
1. CamelCase words in the identifier (e.g. `Resolvedbundle` should be `ResolvedBundle`)
2. Consistently use `Key` suffix to all of the identifiers to clarify that these are to be used for label/annotation keys.
3. Drop `Core` prefix.

Overall, align the identifier name with the value of the constant. e.g.
   - `"core.clusterextension.io/catalog-id"` would map to `CatalogIDKey`
   - `"core.clusterextension.io/package-name"` would map to `PackageNameKey`
   - etc.

Reply:
This is a PoC branch, so it's not going into `main` as-is.

Reply:
https://github.com/operator-framework/operator-controller/pull/765